token/confidential-transfer/elgamal-registry/Cargo.toml

```
@@ -0,0 +1,25 @@
+[package]
+name = "spl-elgamal-registry"
+version = "0.1.0"
+description = "Solana ElGamal Registry"
```

nit: can we spell out this is a program?
```suggestion
description = "Solana ElGamal Registry Program"
```